docs/documentation/platform/kms/hsm-integration.mdx

```diff
@@ -94,165 +94,450 @@ For organizations that work with US government agencies, FIPS compliance is almo
+                PEDTimeout1 = 100000;
+                PEDTimeout2 = 200000;
+                PEDTimeout3 = 20000;
+                KeypairGenTimeOut = 2700000;
+                CloningCommandTimeOut = 300000;
+                CommandTimeOutPedSet = 720000;
+              }
+
+              CardReader = {
+                LunaG5Slots = 0;
+                RemoteCommand = 1;
+              }
+
+              Misc = {
+                # Update the paths to point to the mounted path if your folder structure is different from the one mentioned in the previous step.
+                PluginModuleDir = /hsm-client/plugins;
+                MutexFolder = /hsm-client/lock;
+                PE1746Enabled = 1;
+                ToolsDir = /usr/bin;
+
+              }
+
+              Presentation = {
+                ShowEmptySlots = no;
+              }
+
+              LunaSA Client = {
+                ReceiveTimeout = 20000;
+                # Update the paths to point to the mounted path if your folder structure is different from the one mentioned in the previous step.
+                SSLConfigFile = /hsm-client/etc/openssl.cnf;
+                ClientPrivKeyFile = ./etc/ClientNameKey.pem;
+                ClientCertFile = ./etc/ClientNameCert.pem;
+                ServerCAFile = ./etc/CAFile.pem;
+                NetClient = 1;
+                TCPKeepAlive = 1;
+              }
+
+
+              REST = {
+                AppLogLevel = error
+                ServerName = <REDACTED>;
+                ServerPort = 443;
+                AuthTokenConfigURI = <REDACTED>;
+                AuthTokenClientId = <REDACTED>;
+                AuthTokenClientSecret = <REDACTED>;
+                RestClient = 1;
+                ClientTimeoutSec = 120;
+                ClientPoolSize = 32;
+                ClientEofRetryCount = 15;
+                ClientConnectRetryCount = 900;
+                ClientConnectIntervalMs = 1000;
+              }
+              XTC = {
+                Enabled = 1;
+                TimeoutSec = 600;
+              }
+            ```
+
+            Save the file after updating the paths.
+          </Step>
+
+          <Step title="Creating Persistent Volume Claim (PVC)">
+            You need to create a Persistent Volume Claim (PVC) to mount the HSM client files to the Infisical deployment.
+
+
+            ```bash
+            kubectl apply -f - <<EOF
+            apiVersion: v1
+            kind: PersistentVolumeClaim
+            metadata:
+              name: infisical-data-pvc
+            spec:
+              accessModes:
+                - ReadWriteOnce
+              resources:
+                requests:
+                  storage: 500Mi
+            EOF
+            ```
+            The above command will create a PVC named `infisical-data-pvc` with a storage size of `500Mi`. You can change the storage size if needed.
+
+
+            Next we need to create a temporary pod to upload the HSM client files to the PVC.
```

Comment:
nit: technically, you're copying files into a volume that's mounted to a pod, not directly to the PVC itself. the PVC is just the claim to the storage